### PR TITLE
neofs-adm: allow to transfer containers between networks

### LINKED ISSUE #755: Allow transfering containers between networks
It would be nice if we can easily transfer already created containers to a newly created network.

### PR DESCRIPTION
**Pull request by fyrchik**
Close #755 .

### CODE REVIEW

File: cmd/neofs-adm/internal/modules/morph/root.go [hunk 1]
```diff
@@ -145,4 +167,16 @@ func init() {
...
 	updateContractsCmd.Flags().String(contractsInitFlag, "", "path to archive with compiled NeoFS contracts (default fetched from latest github release)")
+
+	RootCmd.AddCommand(dumpContainersCmd)
+	dumpContainersCmd.Flags().StringP(endpointFlag, "r", "", "N3 RPC node endpoint")
+	dumpContainersCmd.Flags().String(containerDumpFlag, "", "file where to save dumped containers")
+	dumpContainersCmd.Flags().String(containerContractFlag, "", "container contract hash (for networks without NNS)")
+	dumpContainersCmd.Flags().StringSlice(containerIDsFlag, nil, "containers to dump")
+
+	RootCmd.AddCommand(restoreContainersCmd)
+	restoreContainersCmd.Flags().String(alphabetWalletsFlag, "", "path to alphabet wallets dir")
```
Comment (carpawell) [inline cmd/neofs-adm/internal/modules/morph/root.go:178]: is it used for smth in `restoreContainersCmd` command?

File: cmd/neofs-adm/internal/modules/morph/container.go [hunk 2]
```diff
@@ -0,0 +1,335 @@
...
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var errInvalidContainerResponse = errors.New("invalid response from container contract")
+
+func dumpContainers(cmd *cobra.Command, _ []string) error {
+	filename, err := cmd.Flags().GetString(containerDumpFlag)
+	if err != nil {
+		return fmt.Errorf("invalid filename: %w", err)
```
Comment (carpawell) [inline cmd/neofs-adm/internal/modules/morph/container.go:27]: is it the reason for the error?

File: cmd/neofs-adm/internal/modules/morph/container.go [hunk 3]
```diff
@@ -0,0 +1,335 @@
...
+		}
+
+		containers = append(containers, cnt)
+	}
+
+	out, err := json.Marshal(containers)
+	if err != nil {
+		return err
+	}
+	return ioutil.WriteFile(filename, out, 0o660)
```
Comment (carpawell) [inline cmd/neofs-adm/internal/modules/morph/container.go:115]: is that way more preferable for octal nums than `0660`?

**Comment by realloc** [comment]
The main use case for it was the private installation without container creation fee, so I'd propose to rely on option 1 for now.

**Review by alexvanin — APPROVED** [review]

**Pull request merged**